### ISSUE TITLE
NVDA does not accurately identify the e-mail providers listed in Windows 8 mail configuration setup.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2826]
**Reported by Harich on 2012-11-26 22:42**
When I tried to configure my e-mail account using built-in mail client for Windows Eight, NVDA fails to accurately identify e-mail providers listed in the setup configuration option of it. While navigating up and down with the arrow keys to select the appropriate mail provider, e.g. aim, yahoo mail, etc., NVDA fails to read it correctly. It repeats the first mail provider in the list, which is Hotmail, as I navigated through the said list. Consequently, I could not tell as to which e-mail provider I am selecting.

**Comment by leonardder** [comment]
This ticket looks a bit abandoned. ANyone with Windows 8.1 who is able to reproduce this and provide exact steps?

**Comment by Adriani90** [comment]
We do not have any feedback from the initial author for a very long time. In Windows 10 with NVDA 2019.1.1 this is not occuring. So I am closing this for now. If you are still having this problem on Windows 8.1, please comment on it and we can reopen this discussion.
